test-requirements.txt

```diff
@@ -1,7 +1,7 @@
+future
 mock
 pytest
 pytest-cov
-gevent==1.3.6; "PyPy" not in platform_python_implementation
+gevent==20.9.0; "PyPy" not in platform_python_implementation
 pylibmc; sys.platform != 'win32'
 python-memcached
-future
```

Comment:
https://github.com/pinterest/pymemcache/search?q=builtins isn't this still used?

Reply:
It is. I just sorted it to the top of the list (alphabetically).

Reply:
doh